atlas-spec/hacspec_lib/src/rng.rs

```diff
@@ -40,4 +40,9 @@ impl Randomness {
         self.pointer += 1;
         Ok(out & 0x1 == 0x1)
     }
+
+    /// Utility to report how much of the random buffer was consumed so far.
+    pub fn rng_consumed(&self) -> usize {
```

Comment:
This doesn't seem to be used?

Reply:
Right, I've used this to find out how much randomness is actually required in tests and the example, so I can minimize time sampling. I'll remove it, since it's not a required part of any spec.